[DE] Update `People/Tournament committee` (#13642)
ppy/osu-wiki: wiki/People/Tournament_Committee/de.md

---

> tags:

Author: ```suggestion
no_native_review: true
tags:
```

  - TC
  - Contest Committee
  - CC
---

# Turnierkomitee

> Das **Turnierkomitee** ist ein Gremium an erfahrenen sowie etablierten Organisatoren und Helfern in der Turnierszene. Die Mitglieder des Turnierkomitees sind anhand ihres orangenen Nutzertitels und des [Gruppenabzeichens](/wiki/People/User_groups), auf dem `TC` steht, erkennbar. Das Gremium wurde als Reaktion auf die schnell wachsende Community-Turnierszene in osu! am 22. Juli 2021 zum ersten Mal über den Kanal `#tournaments` im [osu! Discord-Server](https://discord.com/invite/ppy) öffentlich bekanntgegeben.

Author: ```suggestion
Das **Turnierkomitee** ist ein Gremium an erfahrenen sowie etablierten Organisatoren und Helfern in der Turnierszene. Die Mitglieder des Turnierkomitees sind anhand ihres orangenen Nutzertitels und des [Gruppenabzeichens](/wiki/People/User_groups) mit der Aufschrift `TC` erkennbar. Das Gremium wurde als Reaktion auf die schnell wachsende Community-Turnierszene in osu! am 22. Juli 2021 zum ersten Mal über den Kanal `#tournaments` im [osu! Discord-Server](https://discord.com/invite/ppy) öffentlich bekanntgegeben.
```

Das Turnierkomitee verwaltet einen [Account auf Twitter](https://twitter.com/osu_tcomm) zur Veröffentlichung verschiedener Aktualisierungen und Benachrichtungen. Beachte, dass dieser Kommunikationsweg eine Ergänzung zu den obigen Quellen ist und vom Kanal `#tournament-news` weitestgehend abgelöst wurde.

### Aufnahme neuer Mitglieder

Neue Mitglieder im Turnierkomitee müssen die folgenden Kriterien erfüllen, bevor eine Einstellung in Erwägung gezogen werden kann:

- mindestens 18 Jahre alt (volljährig)
- keine kürzlichen Regelverstöße

Erfolgreiche Bewerber zeichnen sich durch folgende Stärken aus:

- ausgeprägtes Engagement bei offiziellen Turnieren und/oder in der Community-Turnierszene
- bringt eine einzigartige Sichtweise mit, die dem Komittee bisher gefehlt hat
- umfassende Erfahrung in einem bestimmten Bereich der Turnierszene

> - verhält sich online stets vorbildlich und benimmt sich tadellos

Author: ```suggestion
- verhält sich online stets vorbildlich und benimmt sich respektvoll
```
